archivers/upx-devel/Portfile

```diff
@@ -0,0 +1,44 @@
+# $Id$
+
+PortSystem       1.0
+PortGroup        github 1.0
+
+github.setup     upx upx 5c0340f
+
+name             upx-devel
```

Comment:
upx-devel probably conflicts with upx, doesn't it? If so, add `conflicts upx` to this port, and add `conflicts upx-devel` to the upx port.